packages/lmi/tests/test_rate_limiter.py

```diff
@@ -293,3 +325,64 @@ async def test_embedding_rate_limits(
         )
     else:
         assert estimated_tokens_per_second > 0
+
+
+async def _run_rpm_request_test(
+    is_concurrent: bool, llm_config_w_request_limits: dict[str, Any]
+) -> None:
+    """Run request test with RPM limit.
+
+    Args:
+        is_concurrent: Whether to run requests concurrently.
+        llm_config_w_request_limits: llm config with request configuration parameters.
+    """
+    rate_limit_item = next(
+        iter(llm_config_w_request_limits["config"]["request_limit"].values())
+    )
+    req_request = rate_limit_item.amount
+    req_count = req_request + 1
+
+    messages = [Message.create_message(role="user", text=RATE_LIMITER_PROMPT)]
```

Comment:
```suggestion
    messages = [Message(content=RATE_LIMITER_PROMPT)]
```

Easier just to directly instantiate, and default `role` is already `"user"`